blog.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-A-Dog</title>
+  </head>
+  <body>
+    <header>
+      <img class="logoimg" src="adopt-a-dog-spring2019-master/images/adoptadog-logo.svg" alt="adopt a dog logo"/>
+      <h1 class="headertext">Seattle's premiere dog adoption agency</h1>
```

Comment:
It's good practice to use multiword css classes using what's called kabob casing where it would look like

```
<h1 class="some-random-class-name">Hi there</h1>
```